### PR TITLE
Remove pornography, add fossil fuel extraction, soften nuclear energy

### PR DESCRIPTION
**Pull request by jeznag**
1. Pornography can be ethical employment when produced without coercion.
image

2. Adding statement about fossil fuel extraction

3. Nuclear energy is a very broad brush - Gen IV nuclear reactors are safe and can play a role in moving towards reliable, carbon neutral power

### CODE REVIEW

**Comment by tommaitland** [comment]
@jeznag Thanks for the PR! Some thoughts for discussion:

*Fossil Fuels*
We have it covered in 4.b:

` b) lobbies against, or derives a majority of income from actions that discourage or frustrate: peace, universal access to human rights, peaceful assembly and association (including worker associations), a safe environment, or action to curtail the use of fossil fuels or prevent climate change.`

We added it there since you could have say an energy retailer that is also discouraging action to curtail use of fossil fuels, along with an extractor themselves. Do you think that's enough coverage?

*Nuclear*
Gen IV reactors are still in a research phase, so I'm not sure definition is clear enough for the license (it seems to cover a range of reactors). We could qualify it a little more to something like `nuclear energy (except those with a closed fuel cycle)` which covers the more advanced Gen IV reactors and is the point where nuclear becomes a little closer to a renewable energy in terms of environmental impact.

*Pornography*
This is hard, because there are multiple studies that find that pornography adversely affects the brain, can cause addiction, promotes the objectification of women and create unrealistic expectations on women's appearance. While I agree that pornography is "legitimate work" I think there's scientific evidence to say that it is intrinsically harmful, and defining what's "ethical pornography" is difficult.

I think you could apply the same argument to any of the excluded industries – someone working on an oil rig is legitimate work, but we don't want to support fossil fuel extraction. We do want to protect the rights of oil rig workers as much as we do porn stars (covered in the license already).

I'd love some more opinions too (@chrisjensen) – but what do you think about ☝️ ?

**Comment by jeznag** [comment]
Thanks Tom.

_Fossil fuels:_
I think the current restriction is a bit loose. It seems that an oil company which makes no public stance against renewable energy and no public advocacy for more oil to be used would still be allowed to use the license. See discussion [here](https://github.com/stryker-mutator/stryker/issues/1061#issuecomment-409961544) where this question about oil companies is asked.

_Nuclear:_
Closed cycle sounds good to me. I'll make that change.

_Pornography:_
I'm not an expert here and to be honest, I personally am not a fan of the genre for the link to addiction that you cited. Nonetheless, I've seen a few comments on Medium posts recently which make the same point that Jesse made on Twitter. Maybe we could define ethical pornography as:
- all participants having signed explicit consent waivers
- site has access controls to prevent excess/addictive usage

**Comment by chrisjensen** [comment]
Thanks for the PR @jeznag 

_Fossil Fuels_
I think concerns around indirect contributions (#16 ) were the reasons I originally left fossil fuels out of 4a in the first place.
As long as coal, oil and gas are still widely deployed, then any organisation that uses electricity, cars, trucks or planes in the course of their business could reasonably be considered to "derive a majority of income from actions that support or contribute to" the extraction of fossil fuels.

So I think @archcry's concerns about this being added to 4a are legitimate.

You could argue that 4b covers oil and gas companies even if they don't lobby, as at this point in history, anyone extracting, burning or exploring for new reserves of fossil fuels is frustrating efforts to prevent climate change.

If we wanted to lock this down more, we'd probably need to add a 3rd subsection (ie 4c) that locked that restriction down to just oil & gas companies (and their suppliers/contractors)

_Nuclear_
I'm not sure we should be creating exceptions in the license for technology that doesn't exist yet, outside of research.
Once we grant exceptions and the license is adopted, that cat's out of the bag.
Some bodies have raised questions about weather Gen IV is materially safer than existing technology. I'd be inclined to take a precautionary principle on making exceptions.
It's much easier to loosen a license than tighten it, so I think it would be worth waiting for this technology to develop more before a decision is made.

_Pornography_
I agree with Tom's points.
To be honest, when compiling the list, I didn't give much thought to including pornography, it just came across with the standard ethical screens that I pulled from examples in the ethical investment industry. It's a pretty standard screen applied to ethical things.

In practice, I think those two exceptions would make the license un-adoptable even by "good" players in the industry: I find it hard to see a pornography company deliberately limiting usage (and revenue) from their services just so they can use JWL software, and then still having hanging over them the threat that the creator of that software might one day show up and ask you to show them all your artists consent forms and if you cant locate them all, they might revoke your license/take you to court.

So, again, in the interests of keeping the license simple, I question the value of adding these exceptions.

**Comment by Archcry** [comment]
I just find it hard to judge fossil fuel extraction. It does harm the earth, I agree. But to what extend? Governments are progressively tightening the restrictions on extraction and usage of fossil fuel. So I'd say that restricting those companies from using JWL software is just a stab in the back. Especially because the manufacturing, transport and installation of solar and wind energy also causes pollution. Companies that create energy from fossil fuel are operating within restrictions the government set out for them and might not be that much more harmful than solar and wind energy.

I agree that we need to move away from using fossil fuel but there are already plans to do this. And until we have a reliable alternative to fossil fuel we'll just have to deal with it. Atleast, that's my opinion on the matter.

**Comment by chrisjensen** [comment]
As a general rule we don't want to get into ideological discussions here, but that's designed mainly to head off trolling and @Archcry your question seems genuine to me, so I will provide a brief summary of why I think fossil fuels belong on the list.

As tone is always hard to convey on the internet, this is meant purely to share my perspective. I don't blame you for yours 😃 

That burning fossil fuels and the resulting climate change is destabilising the systems that support life on earth and that we are already seeing severe impacts from this is well established.
The issue has been researched to the point where we now have a global carbon budget: the amount of fossil fuels we could burn before crossing significant thresholds of global warming.

Best estimates say that we may have [just 2 years](https://www.carbonbrief.org/six-years-worth-of-current-emissions-would-blow-the-carbon-budget-for-1-5-degrees) left at current rates of extraction and burning before we exceed the safest threshold.
Examining the assets of publicly listed fossil fuel companies reveals that they count among their assets enough identified fossil fuel reserves to exceed that budget [more than 5 times over](https://www.rollingstone.com/politics/politics-news/global-warmings-terrifying-new-math-188550/).

In order to stay within our 6 year budget, the majority of fossil fuels must stay in the ground.
It should be emphasised that this is 6 years before we face unacceptably high risk of crossing into catastrophic climate collapse scenarios. There is a risk that we may already be there. Even if we aren't, we are today seeing widespread death and destruction that would've been completely avoidable if not for climate change driven by fossil fuels. The death toll today from the combined health impacts of fossil fuels is in the hundreds of thousands.

With this knowledge, fossil fuel extraction is unconscionable.

Global warming came into the public vocabulary 30 years ago, and fossil fuel companies have known about it for [at least 50](https://exxonknew.org/). Despite this knowledge and the knowledge that we cannot even burn what is currently on the books, many are still actively searching for more fossil fuel reserves, and many have engaged in actively deceiving the public and policy makers so as to allow greater profits while jeopardising our future.

There is a carbon footprint to the building of any energy generation system, however the bulk of the lifecycle emissions from a coal, gas or oil plant will come from the energy generation over the decades after it's built, not the construction itself (if the bulk footprint were in the construction, it would mean that the energy required to build them would rival the energy they produce which would make them impractical to build).

In short, if there is anything that should be excluded by an ethical license, it is surely contributing to the collapse of stable climate systems on which human civilisation depends, and any company that is still in the business of digging up or burning fossil fuels today has ignored at least 30 years of warning and has been gambling with humanity's future for profit.
